### DISCUSSION TITLE
Train Lora on papespace.

### DISCUSSION THREAD

**Discussion by Nojarosann** [TheLastBen/fast-stable-diffusion#2501] [Q&A]
Is there a way to train a LoRA using paperspace? I've been trying to look for some more specific pieces of info about that but i can't seem to find anything. It's hard to depend on existing LoRAs even for small stuff.

On google colab there used to be a notebook that i used to train some simple LoRAs but as i can't use colab anymore i don't have any other way to do that.

**Comment by TheLastBen** [comment]
there is a LoRA trainer for SDXL on Paperspace

**Reply by Nojarosann** [reply]
do you have any link fo that? i'm sorry for the inconvenience. Also "SDXL" is that different from Stable diffusion 1.5?

**Reply by TheLastBen** [reply]
SDXL is more advanced and way better, but you need a GPU that has 24GB of VRAM, you'll find the SDXL trainer notebook along with the other paperspace notebooks https://console.paperspace.com/github/TheLastBen/PPS

**Reply by Nojarosann** [reply]
is there anything for hte regular SD? i'm using the notebook you made with a 16GB VM... i don't knwo if can sicced in training a SDXL model or if that would work with my VM
